src/ucp/rndv/proto_rndv.c

```diff
@@ -589,7 +589,7 @@ ucs_status_t ucp_proto_rndv_rts_reset(ucp_request_t *req)
 
 ucs_status_t
 ucp_proto_rndv_ack_init(const ucp_proto_common_init_params_t *init_params,
-                        const char *name, ucs_linear_func_t overhead,
+                        const char *name, double overhead,
```

Comment:
Before that patch we were adding ppln overhead to ATS and since this overhead has linear part we were needed to pass linear function here. After decision to make this ppln overhead part of `ppln_perf` there is no need to get linear function as argument. 

I always though that linear overhead (which depends on the message size) for acknowledge performance looks weird so decided to change this argument to double.